schemas/+behavior/TowersBlock.m

```diff
@@ -16,12 +16,14 @@
 
 classdef TowersBlock < dj.Imported
     properties
-        popRel = behavior.DataDirectory
+        popRel = proj(acquisition.Session, 'level->na_level') * ...
+                 proj(acquisition.SessionStarted, 'session_location->na_location', 'remote_path_behavior_file')
```

Comment:
If you would like to populate for sessions that have entries in either `acquisition.SessionStarted` or `behavior.DataDirectory`, then do this:
```
keySource = acquisition.Session & (acquisition.SessionStarted | behavior.DataDirectory)
```

Reply:
My idea is to substitute the behavior.DataDirectory table (since the path info is already on the SessionStarted table)
When I write 
 keySource = acquisition.Session & acquisition.SessionStarted 

I get this error
Cannot restrict by dependent attribute `session_location`.  It must be projected out or renamed before restriction.
I also got a similar error for level attribute in the acquisition.Session table

Should we rename those fields to prevent this?
I can think of this renaming:
SessionStarted.session_location.  --> SessionStarted.session_started_location
Session.level                                    -->. Session.session_last_level